2017/opmodes/DaisyBlueLaunchAutonomous.java

```diff
@@ -0,0 +1,96 @@
+package opmodes;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+import team25core.DeadReckon;
+import team25core.DeadReckonTask;
+import team25core.FourWheelDirectDriveDeadReckon;
+import team25core.Robot;
+import team25core.RobotEvent;
+import team25core.RunToEncoderValueTask;
+import team25core.SingleShotTimerTask;
+
+/**
+ * FTC Team 25: Created by Katelyn Biesiadecki on 12/2/2016.
+ */
+
+@Autonomous(name = "Daisy: Blue Launch Autonomous", group = "Team25")
+public class DaisyBlueLaunchAutonomous extends Robot
```

Comment:
Technically you don't really need this autonomous anymore because you could roll this all into your main auto above?